admin_scripts/os2displaypc/lts_upgrade_in_place_step_3.sh

```diff
@@ -11,26 +11,29 @@
 #================================================================
 #- IMPLEMENTATION
 #-    version         lts_upgrade_in_place_step_3.sh 0.0.1
-#-    author          Carsten Agger
+#-    author          Carsten Agger, Marcus Funch Mogensen
 #-    copyright       Copyright 2020, Magenta Aps
 #-    license         BSD/MIT
 #-    email           info@magenta.dk
 #-
 #================================================================
 #  HISTORY
+#     2021/03/19 : mfm : Add OS2display migration
 #     2021/01/13 : carstena : Script creation
 #
 #================================================================
 # END_OF_HEADER
 #================================================================
 
+OUR_USER=chrome
 
+set -ex
 
 do-release-upgrade -f DistUpgradeViewNonInteractive >  /var/log/os2borgerpc_upgrade_2.log
 
 
 apt-get update
-apt-get install -y python3-pip || exit 1
+apt-get install -y python3-pip
```

Comment:
I _think_ this `apt`/`dpkg` invocation should run `export DEBIAN_FRONTEND=noninteractive` beforehand, even though stage two asks `dpkg` to disable its conffile handling.

Reply:
I added it now at least. I wonder if it would have the same effect without the `export` keyword?